### ISSUE TITLE
Request for Additional YoloNAS Tutorials and Documentation

### ISSUE THREAD

**Issue by hamedgorji** [Deci-AI/super-gradients#992]
Hello, 

Thanks for your diligent work on the YoloNAS project and for making it open-source. 

One of the most appealing aspects of the YOLOv8 project has been its user-friendly approach. The simplicity of loading models, training, validation, and exporting procedures has made it a popular choice among users. The clear and concise instructions (based on both CLI and Python) have certainly contributed to its popularity.
```
from ultralytics import YOLO

# Create a new YOLO model from scratch
model = YOLO('yolov8n.yaml')

# Load a pretrained YOLO model (recommended for training)
model = YOLO('yolov8n.pt')

# Train the model using the 'coco128.yaml' dataset for 3 epochs
results = model.train(data='coco128.yaml', epochs=3)

# Evaluate the model's performance on the validation set
results = model.val()

# Perform object detection on an image using the model
results = model('https://ultralytics.com/images/bus.jpg')

# Export the model to ONNX format
success = model.export(format='onnx')
```

However, in my recent experience with YoloNAS, while I was able to train and test the models, I found the process of quantization and exporting models to be less straightforward. Given the essential role that these steps play in deploying models, I believe many users like myself would greatly benefit from additional guidance in this area.

Therefore, I would like to respectfully suggest the inclusion of a comprehensive tutorial or guide (CLI and Python) on your website. This guide could detail the process of training a model from scratch using custom datasets, testing it, quantizing it, and finally, exporting it. This would undoubtedly serve as a valuable resource for your users and potentially broaden the adoption of YoloNAS by providing users with a more in-depth understanding of its functionality.

I believe that this addition would significantly enhance the user experience and the overall usability of YoloNAS.

Thank you.

**Comment by shaydeci** [comment]
Hey @hamedgorji ,

Please see our [YoloNAS quantization tutorial](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/qat_ptq_yolo_nas.md).

A notebook for training from scratch on a custom dataset, and exporting it after QAT is in the works, so stay tuned (:
